test/state/mpt_hash.cpp

```diff
@@ -33,4 +34,23 @@ hash256 mpt_hash(const std::unordered_map<address, Account>& accounts)
     }
     return trie.hash();
 }
+
+hash256 mpt_hash_txs(const std::unordered_map<uint64_t, Transaction>& transactions)
+{
+    MPT trie;
+    for (const auto& [tx_index, tx] : transactions)
+        trie.insert(rlp::encode(tx_index), rlp::encode(tx));
+
+    return trie.hash();
+}
+
+hash256 mpt_hash_receipts(const std::unordered_map<uint64_t, TransactionReceipt>& tx_receipts)
```

Comment:
As a remark, the other helpers here overload `mpt_hash`, and the arguments here would not be conflict should we want to keep calling all of them `mpt_hash`.

Reply:
Unfortunately it's ambiguous in tests where we call it with empty `initialiser_list`. We can change test but I don't have any preferences here so decided to stay without changing tests.
`evmone/test/unittests/state_mpt_hash_test.cpp:20:15: error: call to 'mpt_hash' is ambiguous
    EXPECT_EQ(mpt_hash({}), emptyMPTHash);`

Reply:
Change the test. Otherwise, you need to change the API already in use.